v8env/fly/index.js

```diff
@@ -14,7 +15,8 @@ export default function flyInit(ivm, dispatcher) {
     http: require('./http'),
     log: flyLogInit(ivm, dispatcher),
     streams: initStreams(ivm, dispatcher),
-    Image: initImage(ivm, dispatcher),
+		Image: initImage(ivm, dispatcher),
+		CSS: initCSS(ivm, dispatcher),
```

Comment:
I think this is the wrong name for this function.

"`fly.CSS` will minify your css" does not sound intuitive to me. I think we'll probably have different functions under this namespace.

Just spitballing here, but it could be `fly.CSS.removeUnused()` or similar. That would be the functional-way of doing it, but a case could be made to make the API more object-oriented.